integration-tests/src/test/java/oracle/kubernetes/operator/JrfDomainOnPvWlst.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
is this a brand new file? should not have 2018 in there

Reply:
Done